packages/app-headless-cms/src/admin/plugins/fieldRenderers/file/SingleFile.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
```

Comment:
In the future, this might be refactored a bit. 

We should allow easy registration of new file-type renderers via a simple plugin. So, if a user had a special file extension, and wanted to create a different file renderer for it, he should be able to do it.

No time for that now unfortunately. 

I'll make an issue.